### ISSUE TITLE
O'Brien's active compression test

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10996]
**Preferred term label:**
O'Brien's active compression test

**Synonyms**

**Definition (free text, please give PubMed ID)**
Pain in shoulder reproduced when examiner applies downward pressure on the arm while patient resists pressure in the following position: elbow extended, shoulder forward flexed to 90 degrees and 10-15 degrees adducted with palm facing down. This can be compared to the same position but with the palm facing up which should be less painful.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Acromioclavicular (AC) joint injury, Shoulder labrum tear, Adhesive capsulitis, Biceps subluxation

**Your nano-attribution (ORCID)**
https://www.physio-pedia.com/O%27Briens_Test

**Comment by pnrobinson** [comment]
@MickeySegal 
Given the very low sensitivity of this test and the myriad similar tests, I am wondering if we want to have one term for each test? https://www.ncbi.nlm.nih.gov/books/NBK538284/
Maybe it is better to have something like
Positive superior labrum anterior posterior provocative test?

**Comment by MickeySegal** [comment]
I'll check with our Orthopedics curator.  Of the others listed we've used "Speed test"

**Comment by pnrobinson** [comment]
OK, this sounds reasonable! Will do.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001227
- label: Positive O'Brien active compression test
- parents: [Abnormal shoulder physiology (HP:0033129)](https://hpo.jax.org/app/browse/term/HP:0033129)
- pmid: PMID:17936025
- def: Pain in shoulder reproduced when examiner applies downward pressure on the arm while patient resists pressure in the following position: elbow extended, shoulder forward flexed to 90 degrees and 10-15 degrees adducted with palm facing down. This can be compared to the same position but with the palm facing up which should be less painful.
- comment: A positive test may be observed in conditions such as acromioclavicular joint injury, shoulder labrum tear, adhesive capsulitis, and biceps subluxation.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10996
- synonyms:
